Tasks/IISWebAppManagementOnMachineGroup/IISWebAppManagementOnMachineGroup.ps1

```diff
@@ -34,6 +34,10 @@ $appPoolIdentityForWebsite = Get-VstsInput -Name "AppPoolIdentityForWebsite"
 $appPoolUsernameForWebsite = Get-VstsInput -Name "AppPoolUsernameForWebsite"
 $appPoolPasswordForWebsite = Get-VstsInput -Name "AppPoolPasswordForWebsite"
 
+$configureAuthenticationForWebsite = Get-VstsInput -Name "ConfigureAuthenticationForWebsite"
```

Comment:
i am not planning to add as we need to move the current L0 tests to rm-extensions repo. Discussed with Ashish on that he said during that story he would cover.